Tests/JWSRSATests.swift

```diff
@@ -149,8 +128,21 @@ class JWSRSATests: RSACryptoTestCase {
         let verifier = Verifier(verifyingAlgorithm: algorithm, publicKey: publicKeyAlice2048!)
 
         XCTAssertTrue(secondJWS.isValid(for: verifier!))
-        XCTAssertEqual(String(data: jws.header.data(), encoding: .utf8), "{\"alg\":\"\(algorithm.rawValue)\"}")
-        XCTAssertEqual(String(data: jws.payload.data(), encoding: .utf8), "The true sign of intelligence is not knowledge but imagination.")
-    }
+        XCTAssertEqual(String(data: secondJWS.payload.data(), encoding: .utf8), "The true sign of intelligence is not knowledge but imagination.")
```

Comment:
One tiny thing: was it intentional to switch this check to test `secondJWS` instead of `jws`? I'm assuming this is the "same" line as [153 in the previous version](https://github.com/airsidemobile/JOSESwift/pull/198/files#diff-be48fadbbd6c98cbf736aafe1313bb8bL153).

Reply:
Yep intentional as it makes a stronger assertion. The second `secondJWS` is the deserialized version of the first `jws`. With that change that we include the deserialization in the round trip.

Reply:
Ok, looks good to me then 👍